packages/govuk-frontend/src/govuk/components/input/input.yaml

```diff
@@ -399,6 +399,14 @@ examples:
       label:
         text: With value
       value: QQ 12 34 56 C
+  - name: zero value
+    hidden: true
+    options:
+      id: with-zero-value
+      name: with-zero-value
+      label:
+        text: With zero value
+      value: !!int 0
```

Comment:
**question** Is the casting definitely necessary here? My understanding was that YAML should treat this is an integer already.